### PR TITLE
Add cluster-autoscaler update check

### LINKED ISSUE #809: Add a check for cluster autoscaler in find-updates
Add following code snippet to the [script](https://github.com/kinvolk/lokomotive/blob/master/scripts/find-updates.sh) in the components section:

```bash
###########################
# Cluster autoscaler
cd "${workdir}"
current_version=$(grep appVersion ./assets/components/cluster-autoscaler/Chart.yaml | cut -d":" -f2 | sed 's/ //g')

tmpdir=$(mktemp -d)
cd "${tmpdir}"

helm repo add autoscaler https://kubernetes.github.io/autoscaler >/dev/null 2>&1
helm repo update >/dev/null 2>&1
helm fetch --untar --untardir ./ autoscaler/cluster-autoscaler
version=$(grep appVersion "${tmpdir}/cluster-autoscaler/Chart.yaml" | cut -d":" -f2 | sed 's/ //g')

printf "${format}" "cluster-autoscaler" "${current_version}" "${version}"
rm -rf "${tmpdir}"
```

Before adding make sure that following command works without error. Right now it errors like this:

```console
$ helm repo add autoscaler https://kubernetes.github.io/autoscaler
Error: looks like "https://kubernetes.github.io/autoscaler" is not a valid chart repository or cannot be reached: failed to fetch https://kubernetes.github.io/autoscaler/index.yaml : 404 Not Found
```

More info in the upstream: https://github.com/kubernetes/autoscaler/issues/3398.

### PR DESCRIPTION
**Pull request by knrt10**
To be merged after #859 

Closes #809

Signed-off-by: knrt10 <kautilya@kinvolk.io>

### CODE REVIEW

**Review by invidian — COMMENTED** [review]
This should close #809

**Comment by invidian** [comment]
>This should close #809

@knrt10 mind adding `Closes #809` to commit message and PR description?

**Comment by knrt10** [comment]
Updated

File: scripts/find-updates.sh [hunk 1]
```diff
@@ -188,6 +188,14 @@ version=$(grep version "${tmpdir}/velero/Chart.yaml" | cut -d":" -f2 | sed 's/ /
 printf "${format}" "velero" "${current_version}" "${version}"
 rm -rf "${tmpdir}"
 
+###########################
+# cluster-autoscaler
+cd "${workdir}"
+current_version=$(grep version assets/charts/components/cluster-autoscaler/Chart.yaml | cut -d":" -f2 | sed 's/ //g' | sed 's/"//g')
+
+get_latest_release kubernetes/autoscaler
+printf "${format}" "cluster-autoscaler" "cluster-autoscaler-chart-${current_version}" "${version}"
```
Comment (invidian) [inline scripts/find-updates.sh:197]: Hm, I'd strip `cluster-autoscaler-chart-` somehow :/

**Review by invidian — CHANGES_REQUESTED** [review]

File: scripts/find-updates.sh (hunk 1 above)
Comment (knrt10) [reply scripts/find-updates.sh:197]: Cool

**Review by invidian — APPROVED** [review]
LGTM

**Comment by knrt10** [comment]
Merging it, as this is small change and won't affect lokomotive

**Pull request merged**